Internal Server Error: /backend_solicitudes/list_dsc
Traceback (most recent call last):
  File "C:\Users\T0N1\.virtualenvs\djangoProject-rjKEZ1nr\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\T0N1\.virtualenvs\djangoProject-rjKEZ1nr\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\T0N1\.virtualenvs\djangoProject-rjKEZ1nr\Lib\site-packages\django\views\decorators\cache.py", line 54, in _view_wrapper
    response = viewfunc(request, *args, **kw)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\T0N1\.virtualenvs\djangoProject-rjKEZ1nr\Lib\site-packages\django\contrib\auth\decorators.py", line 23, in _wrapper_view
    return view_func(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\T0N1\Desktop\djangoProject\App\views.py", line 621, in backend_solicitudes
    solicitudes_aceptadas = dict(list(Solicitud.return_accepted().items())[-31:])
                                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\T0N1\Desktop\djangoProject\App\models.py", line 564, in return_accepted
    resultados[fecha][tipo] += total
    ~~~~~~~~~~~~~~~~~^^^^^^
KeyError: 'Solicitud-Inscripci¾n'
"GET /backend_solicitudes/list_dsc HTTP/1.1" 500 145
Internal Server Error: /backend_solicitudes/list_dsc
Traceback (most recent call last):
  File "C:\Users\T0N1\.virtualenvs\djangoProject-rjKEZ1nr\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\T0N1\.virtualenvs\djangoProject-rjKEZ1nr\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\T0N1\.virtualenvs\djangoProject-rjKEZ1nr\Lib\site-packages\django\views\decorators\cache.py", line 54, in _view_wrapper
    response = viewfunc(request, *args, **kw)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\T0N1\.virtualenvs\djangoProject-rjKEZ1nr\Lib\site-packages\django\contrib\auth\decorators.py", line 23, in _wrapper_view
    return view_func(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\T0N1\Desktop\djangoProject\App\views.py", line 621, in backend_solicitudes
    solicitudes_aceptadas = dict(list(Solicitud.return_accepted().items())[-31:])
                                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\T0N1\Desktop\djangoProject\App\models.py", line 564, in return_accepted
    resultados[fecha][tipo] += total
    ~~~~~~~~~~~~~~~~~^^^^^^
KeyError: 'Solicitud-Inscripci¾n'
"GET /backend_solicitudes/list_dsc HTTP/1.1" 500 92319
Internal Server Error: /backend_solicitudes/list_dsc
Traceback (most recent call last):
  File "C:\Users\T0N1\.virtualenvs\djangoProject-rjKEZ1nr\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\T0N1\.virtualenvs\djangoProject-rjKEZ1nr\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\T0N1\.virtualenvs\djangoProject-rjKEZ1nr\Lib\site-packages\django\views\decorators\cache.py", line 54, in _view_wrapper
    response = viewfunc(request, *args, **kw)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\T0N1\.virtualenvs\djangoProject-rjKEZ1nr\Lib\site-packages\django\contrib\auth\decorators.py", line 23, in _wrapper_view
    return view_func(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\T0N1\Desktop\djangoProject\App\views.py", line 621, in backend_solicitudes
    solicitudes_aceptadas = dict(list(Solicitud.return_accepted().items())[-31:])
                                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\T0N1\Desktop\djangoProject\App\models.py", line 564, in return_accepted
    resultados[fecha][tipo] += total
    ~~~~~~~~~~~~~~~~~^^^^^^
KeyError: 'Solicitud-Inscripci¾n'
"GET /backend_solicitudes/list_dsc HTTP/1.1" 500 92427
Internal Server Error: /edit_musical_publication/
Traceback (most recent call last):
  File "C:\Users\T0N1\.virtualenvs\djangoProject-rjKEZ1nr\Lib\site-packages\django\db\backends\utils.py", line 105, in _execute
    return self.cursor.execute(sql, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
psycopg2.errors.NotNullViolation: el valor nulo en la columna «gender_id» de la relación «App_musical_publication» viola la restricción de no nulo
DETAIL:  La fila que falla contiene (36, Niña Mia, 979-0-202-00003-8, publications/barcodes/Ni±a_Mia_barcode.png, publications/violin-1617972_1280..jpg, 2019-11-30 00:00:00-05, 2024-06-13 16:49:50.4-04, 8, null, null, 4, 59, 60, 60, 59, , Buena Fe, null).


The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "C:\Users\T0N1\.virtualenvs\djangoProject-rjKEZ1nr\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\T0N1\.virtualenvs\djangoProject-rjKEZ1nr\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\T0N1\.virtualenvs\djangoProject-rjKEZ1nr\Lib\site-packages\django\views\decorators\cache.py", line 54, in _view_wrapper
    response = viewfunc(request, *args, **kw)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\T0N1\.virtualenvs\djangoProject-rjKEZ1nr\Lib\site-packages\django\contrib\auth\decorators.py", line 23, in _wrapper_view
    return view_func(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\T0N1\Desktop\djangoProject\App\views.py", line 1393, in edit_musical_publication
    musical_publication.save()
  File "C:\Users\T0N1\.virtualenvs\djangoProject-rjKEZ1nr\Lib\site-packages\django\db\models\base.py", line 822, in save
    self.save_base(
  File "C:\Users\T0N1\.virtualenvs\djangoProject-rjKEZ1nr\Lib\site-packages\django\db\models\base.py", line 909, in save_base
    updated = self._save_table(
              ^^^^^^^^^^^^^^^^^
  File "C:\Users\T0N1\.virtualenvs\djangoProject-rjKEZ1nr\Lib\site-packages\django\db\models\base.py", line 1036, in _save_table
    updated = self._do_update(
              ^^^^^^^^^^^^^^^^
  File "C:\Users\T0N1\.virtualenvs\djangoProject-rjKEZ1nr\Lib\site-packages\django\db\models\base.py", line 1101, in _do_update
    return filtered._update(values) > 0
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\T0N1\.virtualenvs\djangoProject-rjKEZ1nr\Lib\site-packages\django\db\models\query.py", line 1278, in _update
    return query.get_compiler(self.db).execute_sql(CURSOR)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\T0N1\.virtualenvs\djangoProject-rjKEZ1nr\Lib\site-packages\django\db\models\sql\compiler.py", line 1990, in execute_sql
    cursor = super().execute_sql(result_type)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\T0N1\.virtualenvs\djangoProject-rjKEZ1nr\Lib\site-packages\django\db\models\sql\compiler.py", line 1562, in execute_sql
    cursor.execute(sql, params)
  File "C:\Users\T0N1\.virtualenvs\djangoProject-rjKEZ1nr\Lib\site-packages\django\db\backends\utils.py", line 122, in execute
    return super().execute(sql, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\T0N1\.virtualenvs\djangoProject-rjKEZ1nr\Lib\site-packages\django\db\backends\utils.py", line 79, in execute
    return self._execute_with_wrappers(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\T0N1\.virtualenvs\djangoProject-rjKEZ1nr\Lib\site-packages\django\db\backends\utils.py", line 92, in _execute_with_wrappers
    return executor(sql, params, many, context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\T0N1\.virtualenvs\djangoProject-rjKEZ1nr\Lib\site-packages\django\db\backends\utils.py", line 100, in _execute
    with self.db.wrap_database_errors:
  File "C:\Users\T0N1\.virtualenvs\djangoProject-rjKEZ1nr\Lib\site-packages\django\db\utils.py", line 91, in __exit__
    raise dj_exc_value.with_traceback(traceback) from exc_value
  File "C:\Users\T0N1\.virtualenvs\djangoProject-rjKEZ1nr\Lib\site-packages\django\db\backends\utils.py", line 105, in _execute
    return self.cursor.execute(sql, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
django.db.utils.IntegrityError: el valor nulo en la columna «gender_id» de la relación «App_musical_publication» viola la restricción de no nulo
DETAIL:  La fila que falla contiene (36, Niña Mia, 979-0-202-00003-8, publications/barcodes/Ni±a_Mia_barcode.png, publications/violin-1617972_1280..jpg, 2019-11-30 00:00:00-05, 2024-06-13 16:49:50.4-04, 8, null, null, 4, 59, 60, 60, 59, , Buena Fe, null).

"POST /edit_musical_publication/ HTTP/1.1" 500 160775
Internal Server Error: /edit_musical_publication/
Traceback (most recent call last):
  File "C:\Users\T0N1\.virtualenvs\djangoProject-rjKEZ1nr\Lib\site-packages\django\db\backends\utils.py", line 105, in _execute
    return self.cursor.execute(sql, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
psycopg2.errors.NotNullViolation: el valor nulo en la columna «gender_id» de la relación «App_musical_publication» viola la restricción de no nulo
DETAIL:  La fila que falla contiene (33, Ojala, 979-0-201-00002-2, publications/barcodes/Ojala_barcode.png, publications/Silvio_Rodríguez.jpg, 2013-05-20 00:00:00-04, 2024-06-13 13:54:25.702-04, 5, null, null, 5, 56, 57, 57, 56, Tremendo exito de Silvio, Silvio Rodriguez, null).


The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "C:\Users\T0N1\.virtualenvs\djangoProject-rjKEZ1nr\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\T0N1\.virtualenvs\djangoProject-rjKEZ1nr\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\T0N1\.virtualenvs\djangoProject-rjKEZ1nr\Lib\site-packages\django\views\decorators\cache.py", line 54, in _view_wrapper
    response = viewfunc(request, *args, **kw)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\T0N1\.virtualenvs\djangoProject-rjKEZ1nr\Lib\site-packages\django\contrib\auth\decorators.py", line 23, in _wrapper_view
    return view_func(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\T0N1\Desktop\djangoProject\App\views.py", line 1393, in edit_musical_publication
    musical_publication.save()
  File "C:\Users\T0N1\.virtualenvs\djangoProject-rjKEZ1nr\Lib\site-packages\django\db\models\base.py", line 822, in save
    self.save_base(
  File "C:\Users\T0N1\.virtualenvs\djangoProject-rjKEZ1nr\Lib\site-packages\django\db\models\base.py", line 909, in save_base
    updated = self._save_table(
              ^^^^^^^^^^^^^^^^^
  File "C:\Users\T0N1\.virtualenvs\djangoProject-rjKEZ1nr\Lib\site-packages\django\db\models\base.py", line 1036, in _save_table
    updated = self._do_update(
              ^^^^^^^^^^^^^^^^
  File "C:\Users\T0N1\.virtualenvs\djangoProject-rjKEZ1nr\Lib\site-packages\django\db\models\base.py", line 1101, in _do_update
    return filtered._update(values) > 0
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\T0N1\.virtualenvs\djangoProject-rjKEZ1nr\Lib\site-packages\django\db\models\query.py", line 1278, in _update
    return query.get_compiler(self.db).execute_sql(CURSOR)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\T0N1\.virtualenvs\djangoProject-rjKEZ1nr\Lib\site-packages\django\db\models\sql\compiler.py", line 1990, in execute_sql
    cursor = super().execute_sql(result_type)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\T0N1\.virtualenvs\djangoProject-rjKEZ1nr\Lib\site-packages\django\db\models\sql\compiler.py", line 1562, in execute_sql
    cursor.execute(sql, params)
  File "C:\Users\T0N1\.virtualenvs\djangoProject-rjKEZ1nr\Lib\site-packages\django\db\backends\utils.py", line 122, in execute
    return super().execute(sql, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\T0N1\.virtualenvs\djangoProject-rjKEZ1nr\Lib\site-packages\django\db\backends\utils.py", line 79, in execute
    return self._execute_with_wrappers(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\T0N1\.virtualenvs\djangoProject-rjKEZ1nr\Lib\site-packages\django\db\backends\utils.py", line 92, in _execute_with_wrappers
    return executor(sql, params, many, context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\T0N1\.virtualenvs\djangoProject-rjKEZ1nr\Lib\site-packages\django\db\backends\utils.py", line 100, in _execute
    with self.db.wrap_database_errors:
  File "C:\Users\T0N1\.virtualenvs\djangoProject-rjKEZ1nr\Lib\site-packages\django\db\utils.py", line 91, in __exit__
    raise dj_exc_value.with_traceback(traceback) from exc_value
  File "C:\Users\T0N1\.virtualenvs\djangoProject-rjKEZ1nr\Lib\site-packages\django\db\backends\utils.py", line 105, in _execute
    return self.cursor.execute(sql, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
django.db.utils.IntegrityError: el valor nulo en la columna «gender_id» de la relación «App_musical_publication» viola la restricción de no nulo
DETAIL:  La fila que falla contiene (33, Ojala, 979-0-201-00002-2, publications/barcodes/Ojala_barcode.png, publications/Silvio_Rodríguez.jpg, 2013-05-20 00:00:00-04, 2024-06-13 13:54:25.702-04, 5, null, null, 5, 56, 57, 57, 56, Tremendo exito de Silvio, Silvio Rodriguez, null).

"POST /edit_musical_publication/ HTTP/1.1" 500 161283
Internal Server Error: /backend_solicitudes/list_dsc
Traceback (most recent call last):
  File "C:\Users\T0N1\.virtualenvs\djangoProject-rjKEZ1nr\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\T0N1\.virtualenvs\djangoProject-rjKEZ1nr\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\T0N1\.virtualenvs\djangoProject-rjKEZ1nr\Lib\site-packages\django\views\decorators\cache.py", line 54, in _view_wrapper
    response = viewfunc(request, *args, **kw)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\T0N1\.virtualenvs\djangoProject-rjKEZ1nr\Lib\site-packages\django\contrib\auth\decorators.py", line 23, in _wrapper_view
    return view_func(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\T0N1\Desktop\djangoProject\App\views.py", line 621, in backend_solicitudes
    solicitudes_aceptadas = dict(list(Solicitud.return_accepted().items())[-31:])
                                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\T0N1\Desktop\djangoProject\App\models.py", line 564, in return_accepted
    resultados[fecha][tipo] += total
    ~~~~~~~~~~~~~~~~~^^^^^^
KeyError: 'Solicitud-Inscripci¾n'
"GET /backend_solicitudes/list_dsc HTTP/1.1" 500 145
Internal Server Error: /backend_solicitudes/list_dsc
Traceback (most recent call last):
  File "C:\Users\T0N1\.virtualenvs\djangoProject-rjKEZ1nr\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\T0N1\.virtualenvs\djangoProject-rjKEZ1nr\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\T0N1\.virtualenvs\djangoProject-rjKEZ1nr\Lib\site-packages\django\views\decorators\cache.py", line 54, in _view_wrapper
    response = viewfunc(request, *args, **kw)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\T0N1\.virtualenvs\djangoProject-rjKEZ1nr\Lib\site-packages\django\contrib\auth\decorators.py", line 23, in _wrapper_view
    return view_func(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\T0N1\Desktop\djangoProject\App\views.py", line 621, in backend_solicitudes
    solicitudes_aceptadas = dict(list(Solicitud.return_accepted().items())[-31:])
                                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\T0N1\Desktop\djangoProject\App\models.py", line 564, in return_accepted
    resultados[fecha][tipo] += total
    ~~~~~~~~~~~~~~~~~^^^^^^
KeyError: 'Solicitud-Inscripci¾n'
"GET /backend_solicitudes/list_dsc HTTP/1.1" 500 92030
Internal Server Error: /add_editor/
Traceback (most recent call last):
  File "C:\Users\T0N1\.virtualenvs\djangoProject-rjKEZ1nr\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\T0N1\.virtualenvs\djangoProject-rjKEZ1nr\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\T0N1\.virtualenvs\djangoProject-rjKEZ1nr\Lib\site-packages\django\views\decorators\cache.py", line 54, in _view_wrapper
    response = viewfunc(request, *args, **kw)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\T0N1\.virtualenvs\djangoProject-rjKEZ1nr\Lib\site-packages\django\contrib\auth\decorators.py", line 23, in _wrapper_view
    return view_func(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\T0N1\Desktop\djangoProject\App\views.py", line 957, in add_editor
    editor.prefijo = generate_prefijo_editor(request.POST.get('editorPrefijo'))
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\T0N1\Desktop\djangoProject\App\views.py", line 199, in generate_prefijo_editor
    valor_inicial = Rango_Prefijo_Editor.objects.get(tipo='P-Inferior').rango_inferior + 1
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\T0N1\.virtualenvs\djangoProject-rjKEZ1nr\Lib\site-packages\django\db\models\manager.py", line 87, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\T0N1\.virtualenvs\djangoProject-rjKEZ1nr\Lib\site-packages\django\db\models\query.py", line 649, in get
    raise self.model.DoesNotExist(
App.models.Rango_Prefijo_Editor.DoesNotExist: Rango_Prefijo_Editor matching query does not exist.
"POST /add_editor/ HTTP/1.1" 500 91670
Internal Server Error: /add_musical_publicaton/
Traceback (most recent call last):
  File "C:\Users\T0N1\.virtualenvs\djangoProject-rjKEZ1nr\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\T0N1\.virtualenvs\djangoProject-rjKEZ1nr\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\T0N1\.virtualenvs\djangoProject-rjKEZ1nr\Lib\site-packages\django\views\decorators\cache.py", line 54, in _view_wrapper
    response = viewfunc(request, *args, **kw)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\T0N1\.virtualenvs\djangoProject-rjKEZ1nr\Lib\site-packages\django\contrib\auth\decorators.py", line 23, in _wrapper_view
    return view_func(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\T0N1\Desktop\djangoProject\App\views.py", line 1329, in add_musical_publication
    return render(request, "publicaciones/add_publication.html", data)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\T0N1\.virtualenvs\djangoProject-rjKEZ1nr\Lib\site-packages\django\shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\T0N1\.virtualenvs\djangoProject-rjKEZ1nr\Lib\site-packages\django\template\loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\T0N1\.virtualenvs\djangoProject-rjKEZ1nr\Lib\site-packages\django\template\loader.py", line 15, in get_template
    return engine.get_template(template_name)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\T0N1\.virtualenvs\djangoProject-rjKEZ1nr\Lib\site-packages\django\template\backends\django.py", line 33, in get_template
    return Template(self.engine.get_template(template_name), self)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\T0N1\.virtualenvs\djangoProject-rjKEZ1nr\Lib\site-packages\django\template\engine.py", line 177, in get_template
    template, origin = self.find_template(template_name)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\T0N1\.virtualenvs\djangoProject-rjKEZ1nr\Lib\site-packages\django\template\engine.py", line 159, in find_template
    template = loader.get_template(name, skip=skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\T0N1\.virtualenvs\djangoProject-rjKEZ1nr\Lib\site-packages\django\template\loaders\cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\T0N1\.virtualenvs\djangoProject-rjKEZ1nr\Lib\site-packages\django\template\loaders\base.py", line 28, in get_template
    return Template(
           ^^^^^^^^^
  File "C:\Users\T0N1\.virtualenvs\djangoProject-rjKEZ1nr\Lib\site-packages\django\template\base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
                    ^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\T0N1\.virtualenvs\djangoProject-rjKEZ1nr\Lib\site-packages\django\template\base.py", line 196, in compile_nodelist
    return parser.parse()
           ^^^^^^^^^^^^^^
  File "C:\Users\T0N1\.virtualenvs\djangoProject-rjKEZ1nr\Lib\site-packages\django\template\base.py", line 510, in parse
    raise self.error(token, e)
  File "C:\Users\T0N1\.virtualenvs\djangoProject-rjKEZ1nr\Lib\site-packages\django\template\base.py", line 508, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\T0N1\.virtualenvs\djangoProject-rjKEZ1nr\Lib\site-packages\django\template\loader_tags.py", line 295, in do_extends
    nodelist = parser.parse()
               ^^^^^^^^^^^^^^
  File "C:\Users\T0N1\.virtualenvs\djangoProject-rjKEZ1nr\Lib\site-packages\django\template\base.py", line 510, in parse
    raise self.error(token, e)
  File "C:\Users\T0N1\.virtualenvs\djangoProject-rjKEZ1nr\Lib\site-packages\django\template\base.py", line 508, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\T0N1\.virtualenvs\djangoProject-rjKEZ1nr\Lib\site-packages\django\template\loader_tags.py", line 234, in do_block
    nodelist = parser.parse(("endblock",))
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\T0N1\.virtualenvs\djangoProject-rjKEZ1nr\Lib\site-packages\django\template\base.py", line 510, in parse
    raise self.error(token, e)
  File "C:\Users\T0N1\.virtualenvs\djangoProject-rjKEZ1nr\Lib\site-packages\django\template\base.py", line 508, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\T0N1\.virtualenvs\djangoProject-rjKEZ1nr\Lib\site-packages\django\template\loader_tags.py", line 234, in do_block
    nodelist = parser.parse(("endblock",))
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\T0N1\.virtualenvs\djangoProject-rjKEZ1nr\Lib\site-packages\django\template\base.py", line 510, in parse
    raise self.error(token, e)
  File "C:\Users\T0N1\.virtualenvs\djangoProject-rjKEZ1nr\Lib\site-packages\django\template\base.py", line 508, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\T0N1\.virtualenvs\djangoProject-rjKEZ1nr\Lib\site-packages\django\template\loader_tags.py", line 234, in do_block
    nodelist = parser.parse(("endblock",))
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\T0N1\.virtualenvs\djangoProject-rjKEZ1nr\Lib\site-packages\django\template\base.py", line 510, in parse
    raise self.error(token, e)
  File "C:\Users\T0N1\.virtualenvs\djangoProject-rjKEZ1nr\Lib\site-packages\django\template\base.py", line 508, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\T0N1\.virtualenvs\djangoProject-rjKEZ1nr\Lib\site-packages\django\template\loader_tags.py", line 228, in do_block
    raise TemplateSyntaxError(
django.template.exceptions.TemplateSyntaxError: 'block' tag with name 'Value-Autores' appears more than once
"GET /add_musical_publicaton/ HTTP/1.1" 500 340573
Internal Server Error: /
Traceback (most recent call last):
  File "C:\Users\T0N1\.virtualenvs\djangoProject-rjKEZ1nr\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\T0N1\.virtualenvs\djangoProject-rjKEZ1nr\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\T0N1\Desktop\djangoProject\App\views.py", line 450, in frontend
    return render(request, 'frontend.html')
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\T0N1\.virtualenvs\djangoProject-rjKEZ1nr\Lib\site-packages\django\shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\T0N1\.virtualenvs\djangoProject-rjKEZ1nr\Lib\site-packages\django\template\loader.py", line 62, in render_to_string
    return template.render(context, request)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\T0N1\.virtualenvs\djangoProject-rjKEZ1nr\Lib\site-packages\django\template\backends\django.py", line 61, in render
    return self.template.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\T0N1\.virtualenvs\djangoProject-rjKEZ1nr\Lib\site-packages\django\template\base.py", line 171, in render
    return self._render(context)
           ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\T0N1\.virtualenvs\djangoProject-rjKEZ1nr\Lib\site-packages\django\template\base.py", line 163, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\T0N1\.virtualenvs\djangoProject-rjKEZ1nr\Lib\site-packages\django\template\base.py", line 1000, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\T0N1\.virtualenvs\djangoProject-rjKEZ1nr\Lib\site-packages\django\template\base.py", line 1000, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\T0N1\.virtualenvs\djangoProject-rjKEZ1nr\Lib\site-packages\django\template\base.py", line 961, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\T0N1\.virtualenvs\djangoProject-rjKEZ1nr\Lib\site-packages\django\template\loader_tags.py", line 159, in render
    return compiled_parent._render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\T0N1\.virtualenvs\djangoProject-rjKEZ1nr\Lib\site-packages\django\template\base.py", line 163, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\T0N1\.virtualenvs\djangoProject-rjKEZ1nr\Lib\site-packages\django\template\base.py", line 1000, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\T0N1\.virtualenvs\djangoProject-rjKEZ1nr\Lib\site-packages\django\template\base.py", line 1000, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\T0N1\.virtualenvs\djangoProject-rjKEZ1nr\Lib\site-packages\django\template\base.py", line 961, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\T0N1\.virtualenvs\djangoProject-rjKEZ1nr\Lib\site-packages\django\templatetags\static.py", line 116, in render
    url = self.url(context)
          ^^^^^^^^^^^^^^^^^
  File "C:\Users\T0N1\.virtualenvs\djangoProject-rjKEZ1nr\Lib\site-packages\django\templatetags\static.py", line 113, in url
    return self.handle_simple(path)
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\T0N1\.virtualenvs\djangoProject-rjKEZ1nr\Lib\site-packages\django\templatetags\static.py", line 129, in handle_simple
    return staticfiles_storage.url(path)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\T0N1\.virtualenvs\djangoProject-rjKEZ1nr\Lib\site-packages\django\contrib\staticfiles\storage.py", line 203, in url
    return self._url(self.stored_name, name, force)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\T0N1\.virtualenvs\djangoProject-rjKEZ1nr\Lib\site-packages\django\contrib\staticfiles\storage.py", line 182, in _url
    hashed_name = hashed_name_func(*args)
                  ^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\T0N1\.virtualenvs\djangoProject-rjKEZ1nr\Lib\site-packages\django\contrib\staticfiles\storage.py", line 516, in stored_name
    raise ValueError(
ValueError: Missing staticfiles manifest entry for 'js/chart_utils.js'
"GET / HTTP/1.1" 500 145
Internal Server Error: /add_musical_publicaton/
Traceback (most recent call last):
  File "C:\Users\T0N1\.virtualenvs\djangoProject-rjKEZ1nr\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\T0N1\.virtualenvs\djangoProject-rjKEZ1nr\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\T0N1\.virtualenvs\djangoProject-rjKEZ1nr\Lib\site-packages\django\views\decorators\cache.py", line 54, in _view_wrapper
    response = viewfunc(request, *args, **kw)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\T0N1\.virtualenvs\djangoProject-rjKEZ1nr\Lib\site-packages\django\contrib\auth\decorators.py", line 23, in _wrapper_view
    return view_func(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\T0N1\Desktop\djangoProject\App\views.py", line 1330, in add_musical_publication
    return render(request, "publicaciones/add_publication.html", data)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\T0N1\.virtualenvs\djangoProject-rjKEZ1nr\Lib\site-packages\django\shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\T0N1\.virtualenvs\djangoProject-rjKEZ1nr\Lib\site-packages\django\template\loader.py", line 62, in render_to_string
    return template.render(context, request)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\T0N1\.virtualenvs\djangoProject-rjKEZ1nr\Lib\site-packages\django\template\backends\django.py", line 61, in render
    return self.template.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\T0N1\.virtualenvs\djangoProject-rjKEZ1nr\Lib\site-packages\django\template\base.py", line 171, in render
    return self._render(context)
           ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\T0N1\.virtualenvs\djangoProject-rjKEZ1nr\Lib\site-packages\django\template\base.py", line 163, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\T0N1\.virtualenvs\djangoProject-rjKEZ1nr\Lib\site-packages\django\template\base.py", line 1000, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\T0N1\.virtualenvs\djangoProject-rjKEZ1nr\Lib\site-packages\django\template\base.py", line 1000, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\T0N1\.virtualenvs\djangoProject-rjKEZ1nr\Lib\site-packages\django\template\base.py", line 961, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\T0N1\.virtualenvs\djangoProject-rjKEZ1nr\Lib\site-packages\django\template\loader_tags.py", line 159, in render
    return compiled_parent._render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\T0N1\.virtualenvs\djangoProject-rjKEZ1nr\Lib\site-packages\django\template\base.py", line 163, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\T0N1\.virtualenvs\djangoProject-rjKEZ1nr\Lib\site-packages\django\template\base.py", line 1000, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\T0N1\.virtualenvs\djangoProject-rjKEZ1nr\Lib\site-packages\django\template\base.py", line 1000, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\T0N1\.virtualenvs\djangoProject-rjKEZ1nr\Lib\site-packages\django\template\base.py", line 961, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\T0N1\.virtualenvs\djangoProject-rjKEZ1nr\Lib\site-packages\django\template\loader_tags.py", line 65, in render
    result = block.nodelist.render(context)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\T0N1\.virtualenvs\djangoProject-rjKEZ1nr\Lib\site-packages\django\template\base.py", line 1000, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\T0N1\.virtualenvs\djangoProject-rjKEZ1nr\Lib\site-packages\django\template\base.py", line 1000, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\T0N1\.virtualenvs\djangoProject-rjKEZ1nr\Lib\site-packages\django\template\base.py", line 961, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\T0N1\.virtualenvs\djangoProject-rjKEZ1nr\Lib\site-packages\django\templatetags\static.py", line 116, in render
    url = self.url(context)
          ^^^^^^^^^^^^^^^^^
  File "C:\Users\T0N1\.virtualenvs\djangoProject-rjKEZ1nr\Lib\site-packages\django\templatetags\static.py", line 113, in url
    return self.handle_simple(path)
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\T0N1\.virtualenvs\djangoProject-rjKEZ1nr\Lib\site-packages\django\templatetags\static.py", line 129, in handle_simple
    return staticfiles_storage.url(path)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\T0N1\.virtualenvs\djangoProject-rjKEZ1nr\Lib\site-packages\django\contrib\staticfiles\storage.py", line 203, in url
    return self._url(self.stored_name, name, force)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\T0N1\.virtualenvs\djangoProject-rjKEZ1nr\Lib\site-packages\django\contrib\staticfiles\storage.py", line 182, in _url
    hashed_name = hashed_name_func(*args)
                  ^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\T0N1\.virtualenvs\djangoProject-rjKEZ1nr\Lib\site-packages\django\contrib\staticfiles\storage.py", line 516, in stored_name
    raise ValueError(
ValueError: Missing staticfiles manifest entry for 'multiselect-05/js/jquery.min.js'
"GET /add_musical_publicaton/ HTTP/1.1" 500 145
